src/modelfree/multi/score_worker.py

```diff
@@ -29,6 +29,9 @@ def score_worker(base_config, tune_config, reporter):
     observer = FileStorageObserver.create(osp.join('data', 'sacred', 'score'))
     score_ex.observers.append(observer)
     run = score_ex.run(config_updates=config)
+    index_keys = config.get("index_keys", [])
+
+    idx = {k: v for k, v in config.items()
```

Comment:
The logic of keeping it was to make sure that the required keys would be kept regardless of what was passed in to index_keys. (Which I guess it actually functioning more as additional_index_keys)

I'm not sure how to ensure those things are present without either doing what we're doing, or listing a full hardcoded list of the keys we always want, which doesn't seem particularly less hacky than dynamically generating such a list.

The other alternative, it seems, would just be removing the guardrails altogether and just requiring the user to pass in reasonable things to index_keys, which seems worrying because the logic of what index_keys is doing is so deep in the codebase.